### PR TITLE
#2713 Create sp blitz analysis   query information logged from sp blitz first output tables

### PR DESCRIPTION
**Pull request by Adedba**
#2713 Create sp blitz analysis   query information logged from sp blitz first output tables

### CODE REVIEW

File: sp_BlitzAnalysis.sql [hunk 1]
```diff
@@ -0,0 +1,893 @@
...
+CREATE TABLE #BlitzFirstCounts (
+	[Priority] TINYINT NOT NULL,
+	[FindingsGroup] VARCHAR(50) NOT NULL,
+	[Finding] VARCHAR(200) NOT NULL,
+	[TotalOccurrences] INT NULL,
+	[FirstOccurrence] DATETIMEOFFSET(7) NULL,
+	[LastOccurrence] DATETIMEOFFSET(7) NULL
+);
+
+/* Vallidate variables and set defaults as required */
```
Comment (Withomas) [inline sp_BlitzAnalysis.sql:116]: Please fix small typos ;)

Comment (Adedba) [reply sp_BlitzAnalysis.sql]: Well spotted! thanks for reviewing - I have now corrected :)

**Comment by BrentOzar** [comment]
Howdy sir! You can totally merge these into the dev branch whenever you're comfortable. I don't have the time to review it myself, but if you can write the documentation in readme.md so folks can use it, then by all means, we can put this out in the monthly releases and point 'em to the Slack channel and Github issues if they run into questions.

I'll use it eventually, just trying to limit how much work I add to my plate while I'm in Iceland - we're trying to get out and around and see the sights.

**Comment by Adedba** [comment]
Thanks Brent, no problem at all sounds good! I will get the readme sorted out in the next couple of days and get this included in the merge to Dev.

**Comment by Adedba** [comment]
Right I think that's everything now for a first version, noticed a few little bits whilst doing some extra testing , nothing major but managed to catch them. Will merge this into Dev and hopefully we will get some feedback from the users.

**Pull request merged**